about.html

```diff
@@ -0,0 +1,121 @@
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/js/bootstrap.bundle.min.js" integrity="sha384-w76AqPfDkMBDXo30jS1Sgez6pr3x5MlQ1ZAGC+nuZB+EYdgRZgiwxhTBTkF7CXvN" crossorigin="anonymous" type="module"></script>
+
+    <title>Conference</title>
+</head>
+<body>
+    <headerb>
+        <div class="t-bar">
+            <ul>
+                <li><a href=""><i class="fa-brands fa-facebook-f"></i></a></li>
+                <li><a href="https://twitter.com/TambanaEssohana"><i class="fa-brands fa-twitter"></i></a></li>
+                <li><a href="#">English</a></li>
+                <li><a href="#">MyPage</a></li>
+                <li><a href="#">Logout</a></li>
+            </ul>
+        </div>
+        <nav class="me-container">
+            <div class="logo">
+                <a href="index.html"><img src="images/logo .png" alt="Environment"></a>
+            </div>
+            <div class="my-hamburger">
+            <span class="bar bar1"></span>
+            <span class="bar bar2"></span>
+            <span class="bar bar3"></span>
+                <ul class="my-menus">
+                    <li><a href="index.html" class="text-decoration-none p-2">Home</a></li>
+                    <li><a href="index.html#program" class="text-decoration-none">Program</a></li>
+                    <li><a href="index.html#join" class="text-decoration-none">Join</a></li>
+                    <li><a href="index.html#sponsor" class="text-decoration-none">Sponsor</a></li>
+                    <li><a href="about.html#news" class="text-decoration-none">News</a></li>
+                    <li><a href="about.html#cc" class="text-decoration-none p-2">CC Campaign</a></li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+    <section class="abt" id="abt">
+        <div class="a-head">
+            <h6>"Hello !Sharing World"</h6>
+            <h1>CREATIVE COMMONS GLOBAL SUMMIT 2023</h1>
+            <span class="col-3"></span>
+        </div>
+        <div class="a-paragraph">
+            <p>The Global Summit brings together the community of  
+                experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+                This year, we hope to expand our invitation list including organizations and individuals.</p>
+        </div>
+        <div class="add">
+            <p>Please contact us per Email for nay further questions about CC Global Summit 2015!</p>
+            <address class="">ccsummit2O15@cckorea.org</address>
+        </div>
```

Comment:
Brilliant efforts :clap: 
- [ ] However, I think we are missing a few details regarding the `font-sizes`, `font-weigths`, `background-images` and some spacing using `padding` and `margin`. Let's make a comparison and see where we can improve:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/226868963-07ca0924-0ac3-441f-b785-e9fdb13ca071.png)|![image](https://user-images.githubusercontent.com/99715640/226869043-c2cc7eea-a4a1-4e5d-a191-cbae77b419ad.png)|

**_Please check and fix accordingly :pray: _**

Reply:
@AbayomiOlaoye Please I have a question. Why when I run the test locally, I have this display (the capture below) at home and on your side you have something else? I would like to understand :thinking: 
 ![image](https://user-images.githubusercontent.com/114746721/226886940-a320ebcb-fde1-4027-8e1d-e782dba265ad.png)

Reply:
@GhostEsso , it's probably due to the file path. I'll check it, and get back to you.

Reply:
Yeah, so, like I guessed @GhostEsso , the relative file path is not well configured. There is no `image` folder in your `CSS` folder, so referencing the image should be something like `../images/filename\ .png` . That should make it accessible by any other device in different environments.

Reply:
As for the fonts,  check to be sure that the paths are correct too, or just use [Google Fonts](https://fonts.google.com) to import it into your document. I think it's more efficient this way

Reply:
@AbayomiOlaoye After I finished working on my branch, all the folders and files were there. I did a push and at the remote repository the CSS folder is there, but when I tried to reclone my repository, I didn't have the image folder... I tried to solve that by deleting one folder named vscode. I hope it will be good this time. I will reclone my repository to see if I will have all the folders and files.

Reply:
@AbayomiOlaoye It's working now.